README.md

```diff
@@ -125,7 +167,22 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+1.
+  aladdins_lamp = Lamp.create(wishes_remaining: 3)
+  genie1 = Genie.create(name: "Genie1")
+2.
+  genie :  has_one :lamp
+  lamp :   belongs_to :genie
```

Comment:
What I was looking for here was not defining the general relationship, but specifically connecting the two instances from step 1:

```ruby
genie1.lamp = aladdin_lamp
genie1.save
```